### PR TITLE
Add home-routes for Login, user-routes try-catch on login, CSS framework,  Feature/Navigation Overlay 

### PR DESCRIPTION
**Pull request by ErikPlachta**
### ✅ Re-Fixed issues related to the connection I did in #40 and #43 

> I didn't merge those corrections I made into this branch by mistake. Corrected.

### ✅ Add home-routes for Login 

> This didn't exist so I added it.

### ✅ Add user-routes try-catch on login

> This didn't exist so I added it.

### ✅ Add CSS framework

> This didn't exist so I added it.

> A lot more work to be done here, but getting started. Expectations are that team will change anything/everything, but I needed something to verify the Nav Overlay concept.

### ✅ Add Feature/Navigation Overlay 

- #31 & #29 

> ☝🏼 Related to this discussion/issue. **It's a proof-of-concept design that the team said was worth trying on Thursday.** 

>> ⚠️ ❗ ATM the CSS is included in the `./view/layouts/main.handlebars` file head, because it would NOT work 100% if external.

### CODE REVIEW

File: controllers/api/user-routes.js [hunk 1]
```diff
@@ -116,38 +116,56 @@ router.delete('/:id', withAuth, (req, res) => {
 // login 
 router.post('/login', (req, res) => {
```
Comment (ErikPlachta) [inline controllers/api/user-routes.js:118]: The way this file-change reads is weird, but all I did was put a Try-Catch around it AND a more helpful JSON response.

File: controllers/home-routes.js [hunk 2]
```diff
@@ -48,53 +48,17 @@ router.get('/hero/:id', (req, res) => {
...
-            path: "./home-routes",
-        },
-        response: {
-            status: 404,
-            message: "Rquest failure. Page not found."
-           }
-        })
-    }
-});
+})
```
Comment (ErikPlachta) [inline controllers/home-routes.js:55]: Removed this extra route that was from my seed-template still, and replaced it with the **/login** route for Handlebars.js

**Review by mmlawton15 — APPROVED** [review]
looks good to me!

**Review by NicaVulcan — APPROVED** [review]

**Review by ErikPlachta — COMMENTED** [review]
All good, ready to merge.

**Pull request merged**